src/frontend/src/lib/components/tokens/SkeletonTokenBalance.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	import SkeletonText from '$lib/components/ui/SkeletonText.svelte';
+	import type { TokenUi } from '$lib/types/token';
+
+	export let token: TokenUi;
+</script>
+
+{#if token.balance === undefined}
+	<span class="w-full max-w-[100px] mt-2"><SkeletonText /></span>
+{:else}
+	<slot />
```

Comment:
If balance equals null we display zero?

Reply:
correct, at least for now: I checked the code, and when it is set to `null` is because there was an error with the balance loader and the user receive a toast error warning.

It would make sense to give some more permanent feedback there in such case, I will check with @artkorotkikh-dfinity , but, in any case, I would leave for another PR, WDYT?

Reply:
Sounds good

Reply:
Would it be better to display the same `$-` symbols that we show for SNSes without prices if the balance is null? I know this shouldn’t happen, but displaying 0 might be misleading from the user’s perspective.

<img width="679" alt="image" src="https://github.com/user-attachments/assets/8be3e594-52aa-421e-9dd4-bd600b43e727">

Reply:
I like it: we already do that if there is no exchange rate, so for me is cool

@peterpeterparker WDYT?

Reply:
Makes sense

Reply:
Done with https://github.com/dfinity/oisy-wallet/pull/2352